example/src/main/scala/example/libs/zio/kafka/SocketConsumer.scala

```diff
@@ -0,0 +1,44 @@
+package example.libs.zio.kafka
+
+import zhttp.http._
+import zhttp.service._
+import zhttp.socket._
+import zio._
+import zio.kafka.consumer.{Consumer, ConsumerSettings, Subscription}
+import zio.kafka.serde.Serde
+import zio.stream.ZStream
+import zio.system._
+
+object SocketConsumer extends zio.App {
+  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] = {
+
+    envs.flatMap { env =>
```

Comment:
Use for Comprehension for better readability, add comments to explain what is done in each step.